frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/ReverificationResultService.java

```diff
@@ -83,41 +84,50 @@ public TokenRequest constructTokenRequest(String authCode) {
     }
 
     public TokenResponse sendTokenRequest(TokenRequest tokenRequest) {
-        try {
-            LOG.info("Sending IPV token request");
-            int count = 0;
-            int maxTries = 2;
-            TokenResponse tokenResponse;
-
-            do {
-                if (count > 0) {
-                    LOG.warn("Retrying IPV access token request");
-                }
-
-                count++;
+        LOG.info("Sending IPV token request");
+        int count = 0;
+        TokenResponse tokenResponse =
+                new TokenResponse() {
+                    @Override
+                    public boolean indicatesSuccess() {
+                        return false;
+                    }
+
+                    @Override
+                    public HTTPResponse toHTTPResponse() {
+                        return null;
+                    }
+                };
+
+        while (count < MAX_RETRIES) {
```

Comment:
Agreed.  Discussed this with Dan as there are several changes we need to make regarding number of retries, what we retry and whether we should include a wait between retries.  This will be addressed on a different change.